### DISCUSSION TITLE
Option to allow trades to go to 1 emerald.

### DISCUSSION THREAD

**Discussion by MenKlash** [PurpurMC/Purpur#1263] [__Dismissed__] [answered]
The first time I cured a zombie villager, the discount was applied correctly. However, the price, with the previous discount, would stay forever, **no matter how many times I cured him**.

I searched in the purpur.yml to see if I could change this behaviour, but I didn't find anything related to that.

Then, I investigated if paper had an option to enable it, and I found out that in paper.yml, there's an option called "**fix-curing-zombie-villager-discount-exploit**" that disables this "exploit".

Purpur is a fork of Paper, so I think you should be able to add this option without much trouble, but I don't know tho.

**Comment by encode42** [comment, marked as answer]
Purpur is indeed a fork of Paper, meaning everything Paper has, Purpur has. Change the aforementioned value in Paper's config and it'll be applied.

**Reply by MenKlash** [reply]
Thanks. I didn't put much attention in the folder "Config" in the files of my server.
